processor/metricstransformprocessor/README.md

```diff
@@ -103,34 +104,31 @@ operations:
 # aggregate away everything but `state` using summation
 ...
 operations:
-  -action: aggregate_labels
-   label_set: [ state ]
-   aggregation_type: sum
+  - action: aggregate_labels
+    label_set: [ state ]
+    aggregation_type: sum
 ```
 
 ### Aggregate Label Values
 ```yaml
 # combine slab_reclaimable & slab_unreclaimable by summation
 ...
 operations:
-  -action: aggregate_label_values
-   label: state
-   aggregated_values: [ slab_reclaimable, slab_unreclaimable ]
-   new_value: slab 
-   aggregation_type: sum
+  - action: aggregate_label_values
+    label: state
+    aggregated_values: [ slab_reclaimable, slab_unreclaimable ]
+    new_value: slab 
+    aggregation_type: sum
 ```
 
 ### Add a label to an existing metric
 ```yaml
-transforms:
 ...
-# The following will append label {Key: `mylabel`, Description: `myvalue`} to the metric `some_name`.
-  - metric_name: some_name
-      action: update
-      operations:
-        - action: add_label
-          new_label: mylabel
-          new_value: myvalue
+# add label `version` with value `opentelemetry collector v0.0.0` to all points
+operations:
+  - action: add_label
+    new_label: version
+    new_value: opentelemetry collector {{version}}
```

Comment:
nit: Ideally there should be a way to escape this and be able to pass the `{{version}}` string as is (without replacement).
Though in practice I doubt anyone would ever run into this limitation.

Reply:
I guess it's possible that some other processor also does substitution for `{{version}}`, but in a different way / with a different value. In that case the ordering of processors will matter.
Still don't think this is a real issue though.

Reply:
Yea this is a very simple substitution so I only defined trivial semantics. Note it only applies the configured values themselves. I may come back and add something more sophisticated later.